recipes/cdutil/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "cdutil" %}
+{% set version = "2.8.2" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/UV-CDAT/cdutil/archive/{{ version }}.tar.gz
+    sha256: a9eb7e4805a9b9f1acc46b84e2e241260e9690ca76cba5331eba2631ec9403b1
+    
+
+build:
+  number: 0
+  skip: True  # [win or py3k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy x.x
+    - genutil
+  run:
+    - python
+    - cdms2
+    - numpy x.x
```

Comment:
Does this package require that a different version be built for different versions of numpy?